components/layout/partials/Logo.tsx

```diff
@@ -14,8 +13,10 @@ export default function Logo(logoProp: LogoProp): JSX.Element {
   return (
     <div className={classes}>
       <h1 className="m-0">
-        <Link to="/">
-          <Image src={logo} alt="Open" width={32} height={32} />
+        <Link href="/">
+          <a>
+            <Image src={"images/logo.png"} alt="Open" width={32} height={32} />
```

Comment:
This <Image/> should be the next/image component rather than our own custom Image component. If we use their react component, we get automatic image resizing and compression and other optimizations for server-speed.

Reply:
Will create a new issue for this one (see #42)